srcpkgs/tuxedo-keyboard-dkms/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'tuxedo-keyboard-dkms'
+pkgname=tuxedo-keyboard-dkms
+version=3.0.9
+revision=1
+archs="x86_64 x86_64-musl"
```

Comment:
I believe `x86_64*` is preferred.

Reply:
Fixed, pushed and verified that it still builds